salesforce/latest/using/community.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Community Sites
+---
+
+To share content externally to a community site and retrieve a shared case as a community member you need to peform a series of actions. Sharing sites externally can be done using either the [Lightning Experience](#lightning-experience) or [Salesforce Classic](#salesforce-classic). [Retrieving shared cases](#retrieving-shared-cases) as a community member can only be done using the Lightning Experience. Before you begin ensure you have added the administrator user of Alfresco Content Services that you use as an administrator of Salesforce to the `GROUP_SALESFORCE_MODERATORS` group in Alfresco Content Services, for more see [Configure a Salesforce Community site]({% link salesforce/latest/config/index.md %}#configure-a-salesforce-community-site).
+
+> **Note:** Support for Salesforce Community is only available when using Alfresco Cloud (Digital Business Platform-as-a-Service (PaaS)).
+
+## Lightning Experience
+
+1. See [Create an Experience Cloud Site pg. 51](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+2. See [Add Members to Your Experience Cloud Site pg. 54](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+3. [Share content externally to community members Lightning Experience](#share-content-externally-to-community-members-lightning-experience)
+
+## Share content externally to community members Lightning Experience
+
+1. Log in to Salesforce.
+
+2. Log in as a community administrator to Alfresco Share in the Canvas app.
```

Comment:
Remove word "community". 
"Log in as an administrator to Alfresco Share in the Canvas app."

Reply:
done